docs/admin/predeployment_check.md

```diff
@@ -33,24 +33,29 @@ $ sudo reboot
     'disable_vfio' in the [server config file](https://github.com/daos-stack/daos/blob/master/utils/config/daos_server.yml#L109),
     but note that this will require running daos_server as root.
 
-!!! note
-	If VFIO is not enabled, you will run into the issue described in:
-	https://github.com/spdk/spdk/issues/1153
+!!! warning
+    If VFIO is not enabled, you will run into the issue described in:
+    https://github.com/spdk/spdk/issues/1153
 
-	When using RHEL 8.1 with official kernel from distribution (4.18.0-147.el8.x86_64)
-	it is not possible to bind nvme devices to uio_pci_generic driver and due to that
-	to use them within SPDK environment:
+    When using RHEL 8.1 with official kernel from distribution (4.18.0-147.el8.x86_64)
+    it is not possible to bind nvme devices to uio_pci_generic driver and due to that
```

Comment:
RHEL 8.1 is no longer supported ... drop this paragraph?

Reply:
It sounds like UIO does not work on RHEL8 & derivatives. Waiting for confirmation.